src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -123,6 +126,27 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
     }
 
     /**
+     * Set key listeners for handling keyboard shortcuts.
+     */
+    private void setKeyListeners() {
+        getRoot().addEventFilter(KeyEvent.KEY_PRESSED, event -> {
+            switch (event.getCode()) {
```

Comment:
Will a switch case work for stuff like `ctrl + something`?

Reply:
Good point, should be changing this later!